.circleci/config.yml

```diff
@@ -24,10 +24,17 @@ jobs:
     steps:
       - attach_workspace:
           at: ~/project
+      - restore_cache:
+          keys:
+            - woopagarme-{{ .Branch }}-{{ checksum "composer.json" }}
```

Comment:
Tava pensando e dado que o parametro `Branch` vai gerar um cache pra cada novo branch que subirmos, na prática não vamos estar aproveitando a feature. Dado que a gente não deve mudar muito acho que podemos remover o parâmetro e aproveitar o cache em builds diferentes e, sempre que for preciso invalidar o cache por algum motivo a gente muda a chave.